src/fides/api/db/samples.py

```diff
@@ -122,7 +120,7 @@ def load_sample_connections_from_project() -> List[SampleConnection]:
     return valid_connections
 
 
-def load_sample_yaml_file(file: TextIO, expand_vars: bool = True) -> Dict:
+def load_sample_yaml_file(file: IO, expand_vars: bool = True) -> Dict:
```

Comment:
What was the reason for this change?

Reply:
mypy I believe -

Reply:
upgrading mypy flagged hundreds of places -